todo_app/tests/test_app_client.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from dotenv import load_dotenv, find_dotenv
+import os
+import requests
+
+from todo_app.classes.view_model import ViewModel
+from todo_app.classes.item_class import Item
+from todo_app import app
+
+@pytest.fixture
+def client():
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+
+    test_app = app.create_app()
+
+    with test_app.test_client() as client:
+        yield client
+
+class StubResponse():
+    def __init__(self, fake_response_data):
+        self.fake_response_data = fake_response_data
+
+    def json(self):
+        return self.fake_response_data
+    
+def stub(url, data, headers, verify):
+    trello_boards_url       = "https://api.trello.com/1/boards/"
+    trello_board_id         = os.environ.get('TRELLO_BOARD_ID','')
+    trello_api_key          = os.environ.get('TRELLO_API_KEY','')
+    trello_api_token        = os.environ.get('TRELLO_API_TOKEN','')
+    trello_api_security     = "key="+trello_api_key+"&token="+trello_api_token
+
+    get_item_req_url = trello_boards_url+trello_board_id+'/cards/?'+trello_api_security 
+    if url == get_item_req_url:
+        fake_response_data = [{
+            'id': 'test_id_1',
+            'name': 'test_todo_item',
+            'idList': '658400a6795546f35c80c673'
```

Comment:
Minor, but these will be a little more resilient if we take this from the environment rather than hard-coding, e.g.:
```suggestion
            'idList': os.getenv('TRELLO_BOARD_LIST_ID_TODO')
```